src/translator/CMakeLists.txt

```diff
@@ -13,10 +13,15 @@ add_library(bergamot-translator STATIC
     sentence_ranges.cpp
     service.cpp
 )
+if (USE_WASM_COMPATIBLE_MARIAN)
+  # Using wasm compatible marian should include this compile definition;
+  # Has to be done here because of the marian's cmake practices
+  target_compile_definitions(bergamot-translator PUBLIC WASM)
+endif()
```

Comment:
Remove this block.

Reply:
The combination of `USE_WASM_COMPATIBLE_MARIAN` (which I think is meant to be a native build that's reduced?) and then defining `WASM` doesn't make sense to me.  To me the define `WASM` means the compilation environment is web assembly (since you've asked to deprecate `__EMSCRIPTEN__`) and it would be fair game to use things that /only/ work in WebAssembly, such as the wormhole.  

If you want to have an in-between Marian that has less baggage and compiles natively, fine, but then be careful about what macro does what.